src/jvmTest/kotlin/org/treeWare/model/operator/DifferenceTests.kt

```diff
@@ -17,18 +18,45 @@ class DifferenceTests {
         val jsonInput1 = readFile("model/operator/difference/mini_test_book_1.json")
         val jsonInput2 = readFile("model/operator/difference/mini_test_book_2.json")
         val expectedCreateTestOutput = "model/operator/difference/mini_test_book_difference_create_1_2.json"
+        val expectedUpdateTestOutput = "model/operator/difference/mini_test_book_difference_update_1_2.json"
         assertNotEquals(jsonInput1, jsonInput2)
 
         val input1 = getMainModelFromJsonString(addressBookMetaModel, jsonInput1)
         val input2 = getMainModelFromJsonString(addressBookMetaModel, jsonInput2)
         val output = difference(input1, input2)
-        val addOutput = output.createModel
+        val createOutput = output.createModel
         val deleteOutput = output.deleteModel
+        val updateOutput = output.updateModel
 
-        assertNotNull(addOutput)
-        assertMatchesJson(addOutput, expectedCreateTestOutput, EncodePasswords.ALL)
+        assertNotNull(createOutput)
+        assertMatchesJson(createOutput, expectedCreateTestOutput, EncodePasswords.ALL)
 
         assertNull(deleteOutput)
+
+        assertNotNull(updateOutput)
+        assertMatchesJson(updateOutput, expectedUpdateTestOutput, EncodePasswords.ALL)
+    }
+
+    @Test
+    fun `Difference operator Create and Update union with old tree result in new tree`() {
```

Comment:
nit: state what the unit-under-test "must" do. For example, this test could be:

The union of the old, create, and update models must result in the new model